flake.nix

```diff
@@ -12,92 +12,216 @@
     };
   };
 
-  outputs = { self, nixpkgs, utils, gomod2nix }:
-    utils.lib.eachSystem [
+  outputs = {
+    self,
+    nixpkgs,
+    utils,
+    gomod2nix,
+  }:
+    {
+      overlays.default = final: prev: {
+        tclip = self.packages."${prev.system}".tclip;
```

Comment:
If I look at other examples https://github.com/ryantm/agenix they are structured differently; t
the overlay defines the packages and then packages attribute just pull it out of nixpkgs itself.

This goes in the other direction; not sure how "wrong" that is.

Reply:
I don't have any strong feelings here, as long as it works well and doesn't break or complicate anything.